File: Visual Studio Product Team/Visual Studio 2010 SDK Samples/60968-Visual Studio 2010 SDK Samples/Code Sweep/C#,C++/VsPackage/Guids.cs
/***************************************************************************

Copyright (c) Microsoft Corporation. All rights reserved.
This code is licensed under the Visual Studio SDK license terms.
THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.

***************************************************************************/

// MUST match guid.h (for the satellite DLL)
using System;

namespace Microsoft.Samples.VisualStudio.CodeSweep.VSPackage
{
    static class GuidList
    {
        public static readonly Guid guidVSPackagePkg = new Guid("{2b621c1e-60a3-48c5-a07d-0ad6d3dd3417}");
        public static readonly Guid guidVSPackageCmdSet =        new Guid("{d0882566-3d01-4578-b4f2-0aff36119700}");
    };
}

File: Visual Studio Product Team/Visual Studio 2010 SDK Samples/[C++]-Visual Studio 2010 SDK Samples/Code Sweep/C++,VB/VsPackage/Guids.vb
'**************************************************************************

'Copyright (c) Microsoft Corporation. All rights reserved.
'This code is licensed under the Visual Studio SDK license terms.
'THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
'ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
'IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
'PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.

'**************************************************************************

' MUST match guid.h (for the satellite DLL)

Imports Microsoft.VisualBasic
Imports System

Namespace Microsoft.Samples.VisualStudio.CodeSweep.VSPackage
    Public Class GuidStrings
        Public Const GuidVSPackagePkg As String = "32E9CE10-DFC9-4b7d-B22A-396DC0B8F6E3"
        Public Const GuidVSPackageCmdSet As String = "4D0C001B-DA7A-4db9-BA55-C76CB3105EEA"
    End Class

    Friend Class GuidList
        Private Sub New()
        End Sub

        Public Shared ReadOnly guidVSPackagePkg As New Guid(GuidStrings.GuidVSPackagePkg)
        Public Shared ReadOnly guidVSPackageCmdSet As New Guid(GuidStrings.GuidVSPackageCmdSet)
    End Class
End Namespace